lib/internal/Magento/Framework/Locale/Config.php

```diff
@@ -74,7 +74,7 @@ class Config implements \Magento\Framework\Locale\ConfigInterface
         'lv_LV', /*Latvian (Latvia)*/
         'mk_MK', /*Macedonian (Macedonia)*/
         'mn_Cyrl_MN', /*Mongolian (Mongolia)*/
-        'ms_Latn_MY', /*Malaysian (Malaysia)*/
+        'ms_MY', /*Malaysian (Malaysia)*/
```

Comment:
I [doubt](https://www.google.com/search?ei=6TK2Wr_5E4b-sQHlj4rwBw&q=locale+ms_Latn_MY&oq=locale+ms_Latn_MY&gs_l=psy-ab.3...10031.10575.0.10662.7.6.0.0.0.0.186.446.2j2.4.0....0...1c.1.64.psy-ab..6.0.0....0.ZHbjyLSAfJM) this is a typo, should we add both locale codes maybe?

Reply:
I will add in a minute ;-)

Reply:
Done!